corehq/apps/cloudcare/templates/formplayer/debugger.html

```diff
@@ -104,9 +104,9 @@
     <!--/ko-->
     <!--ko if: xPathQuery.isTraceTruncated() && options.showInFull-->
     <div>
-        <i>Showing the first </i><i data-bind="text: xPathQuery.getMaxLines()"/><i> lines...</i>
+        <em>{% trans "Showing the first" %} <span data-bind="text: xPathQuery.getMaxLines()"/>{% trans " lines..." %}</em>
```

Comment:
nit, but having the leading space and the ellipsis in the translate tag will mean that the translator will have to add those into the translation too which is a bit awkward.

Reply:
Fixed. I left the "..." because I think that's part of the translated string, but agree WRT the spaces.